CSS/styles.css

```diff
@@ -0,0 +1,882 @@
+}
+
+.show_more {
+  display: block;
+  transition-duration: 1s;
+}
+
+.speaker_1-image img {
+  width: 6.7rem;
+  height: 6.7rem;
+  margin-left: 12.54px;
+}
+
+.speaker_2-image img {
+  width: 6.5rem;
+  height: 6.7rem;
+  margin-left: 12.54px;
+}
+
+.info h4 {
+  font-weight: 700;
+  font-size: 15px;
+  line-height: 18px;
+  letter-spacing: -0.02em;
+  color: black;
+}
+
+.info .sub-text {
+  font-weight: 700;
+  font-size: 10px;
+  line-height: 15px;
+  align-items: center;
+  letter-spacing: -0.02em;
+  color: var(--secondary);
+  width: 14.125rem;
+}
+
+.info .caption {
+  font-weight: 700;
+  font-size: 9px;
+  line-height: 14px;
+  letter-spacing: -0.06em;
+  color: #000;
+  width: 14.125rem;
+}
+
+.more {
+  width: 93.6%;
+  height: 2.4rem;
+  border: 1px solid #dcdcdc;
+  filter: drop-shadow(0.125rem 0.125rem 0.25rem rgba(0, 0, 0, 0.25));
+  border-radius: 0.25rem;
+  -webkit-filter: drop-shadow(0.125rem 0.125rem 0.25rem rgba(0, 0, 0, 0.25));
+}
+
+.arrow {
+  margin-left: 0.6rem;
+}
+
+.footer_ {
+  display: none;
+}
+
+#partner {
+  display: none;
+}
+
+/* About Us Section */
+
+#about_hero_container {
+  display: flex;
+  background: url(../Backgrounds/mobile_intro_bg.png), #f7f7f8;
+  background-position: 100%;
+  background-repeat: no-repeat;
+  background-size: contain;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+  padding-top: 0.313rem;
+  gap: 0.625rem;
+}
+
+.about_hero h1 {
+  margin: 0;
+  color: transparent;
+  font-size: 2rem;
+  font-weight: 900;
+  background: url(../Backgrounds/text-clip.jpg);
+  background-clip: text;
+  -webkit-background-clip: text;
+  background-position: 100%;
+}
+
+.frame_1 {
+  gap: 0.3125rem;
+  padding: 2px;
+}
+
+small {
+  font-weight: 800;
+  font-size: 0.75rem;
+  line-height: 1.56rem;
+  text-align: center;
+  letter-spacing: 0.02em;
+  text-decoration-line: underline;
+  color: #5f5f5f;
+}
+
+.creative_corner {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+  padding: 2px;
+}
+
+.about_logo {
+  width: 90%;
+}
+
+.about_logo img {
+  object-fit: cover;
+  width: 100%;
+  height: 8.86rem;
+}
+
+.summits {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+  padding: 0;
+  background: #fff;
+  border-bottom: 1px solid #fafafa;
+
+  /* box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25); */
+  margin-bottom: 20px;
+}
+
+.summit_headline {
+  font-weight: 700;
+  font-size: 1.25rem;
+  line-height: 1.75rem;
+  letter-spacing: -0.02em;
+  color: #2c2c2c;
+}
+
+.summit_text {
+  width: 85.3%;
+  font-weight: 700;
+  font-size: 0.75rem;
+  line-height: 1.26rem;
+  letter-spacing: -0.03em;
+  color: #838383;
+}
+
+.previous_summit {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 89.1%;
+  padding: 0;
+  gap: 1.5625rem;
+}
+
+.event_1 {
+  width: 89%;
+}
+
+.event_1 img {
+  object-fit: cover;
+  width: 100%;
+  height: 11.4rem;
+}
```

Comment:
- [ ] To make your design look accurate like that of the original one and also create a good user experience while adhering to the project requirements kindly consider below suggestions:
  -  [ ] Your image should have 2 contents `(year and title)`
  - [ ] The content of each image should be overlayed on the image with a background color matching that of the original design. See below for more details:
  <table>
<tr>
<th align=“middle”>Behance design</th>
<th align=“middle”>Your design</th>
</tr>
<tr>
<td>
<img width="459" alt="Screenshot 2023-06-29 at 01 20 15" src="https://github.com/As1imwe-Mark/Web-Camp-Capstone-project/assets/107192076/e69a63ab-b41f-479d-a46c-26ef0c9242d0">
</td>
<td>
<img width="372" alt="Screenshot 2023-06-29 at 01 19 21" src="https://github.com/As1imwe-Mark/Web-Camp-Capstone-project/assets/107192076/3b5d97b0-581d-4813-9e35-30334a58e481">
</td>
</tr>
</table>

### Desktop
<table>
<tr>
<th align=“middle”>Behance design</th>
<th align=“middle”>Your design</th>
</tr>
<tr>
<td>
<img width="871" alt="Screenshot 2023-06-29 at 01 19 51" src="https://github.com/As1imwe-Mark/Web-Camp-Capstone-project/assets/107192076/fa162179-9b90-419d-b795-be065dd3b0f0">
</td>
<td>
<img width="885" alt="Screenshot 2023-06-29 at 01 20 35" src="https://github.com/As1imwe-Mark/Web-Camp-Capstone-project/assets/107192076/45184a74-6190-48cc-9da9-c88a153a96f4">
</td>
</tr>
</table>